src/Entities/Pawn.cpp

```diff
@@ -377,7 +376,6 @@ void cPawn::HandleFalling(void)
 			if (j == 0)
 			{
 				IsFootInWater |= IsBlockWater(BlockType);
-				IsFootInLiquid |= IsFootInWater || IsBlockLava(BlockType) || (BlockType == E_BLOCK_COBWEB);  // okay so cobweb is not _technically_ a liquid...
```

Comment:
I think we should be doing something with this, rather than just deleting the LOC...

Looks like it's meant to be used in the code below, but somebody forgot.

Reply:
It can always get added back, it's not a complicated piece of code.

Reply:
> Player sometimes receive damage upon falling in cobwebs

possibly causing part of #5323